week05/errors_test.go

```diff
@@ -0,0 +1,92 @@
+package demo
+
+import (
+	"fmt"
+	"testing"
+)
+
+func Test_tablenotfound(t *testing.T) {
```

Comment:
Try to use clearer and more descriptive test names.

Go uses Camel/PascalCase for name.

```suggestion
func Test_ErrTableNotFound(t *testing.T) {
```